src/ResourceManager/OperationalInsights/AzureRM.OperationalInsights.psd1

```diff
@@ -145,8 +146,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Add support for online help
-    - Run Get-Help with the -Online parameter to open the online help in your default Internet browser'
+        ReleaseNotes = 'https://github.com/Azure/azure-powershell/blob/dev/ChangeLog.md'
```

Comment:
You don't need to update the version or the release notes.

You will also need to add the new dataplane sdk assembly into the 'requiredassemblies' proeprty in this psd1